src/main/java/net/imagej/ops/filter/FilterNamespace.java

```diff
@@ -500,7 +503,27 @@
 				performInputFFT, performKernelFFT);
 		return result;
 	}
+	
+	// -- directional derivative --
+	
+	/** Executes the "directional derivative" operation on the given arguments */
+	@OpMethod(op = net.imagej.ops.filter.derivative.PartialDerivativeRAI.class)
+	public <T extends RealType<T>> RandomAccessibleInterval<T> directionalDerivative(final RandomAccessibleInterval<T> in, final int dimension) {
```

Comment:
This one and the next method are a left-over from the rename to `partialDerivative`, I guess? If so, they can now be removed.